flyway/sql/V2___updated_schema.sql

```diff
@@ -0,0 +1,25 @@
+-- Step 1: Add new columns (Null possible initially)

```

Comment:
I lost the data in my DB after migration.
I ran the main branch and manually added new row into ```addresses```. 
<img width="869" alt="Screenshot 2024-12-26 at 9 37 47" src="https://github.com/user-attachments/assets/bbfc64bb-efe5-4fb7-9cac-347989a1f993" />

Then I pulled your branch and ran it. In the ```addresses``` table there is no previously added row.
<img width="869" alt="Screenshot 2024-12-26 at 9 38 52" src="https://github.com/user-attachments/assets/139af184-d7eb-4734-b88a-bf31f025f32a" />